src/celimage/dds.cpp

```diff
@@ -144,7 +144,6 @@ Image* LoadDDSImage(const fs::path& filename)
     }
 
     char header[4];
-    in.read(header, sizeof header);
```

Comment:
we should be more careful when reviewing changes

Reply:
Agreed. Apologies for the breakages along the way, but I do think it's important for an application that's dealing with user content to be a lot stricter about error checking the input files these days.